### ISSUE TITLE
ActiveModel complains about missing attributes when using select on Storext model

### ISSUE THREAD

**Issue by jpmoral** [G5/storext#21]
```
class Foo < ActiveRecord::Base
  include Storext.model

  store_attributes :settings do
    attribute_name String
  end
end

Foo.where(some_attr: some_value).select(:some_attr).first
# => ActiveModel::MissingAttributeError: missing attribute: settings
```

**Comment by ramontayag** [comment]
Fixed in 2.0

https://github.com/G5/storext/blob/master/CHANGELOG.md#200

**Comment by ramontayag** [comment]
@jpmoral you're right - I'm mistaken. This particular issue was not fixed. Re-opening.
